framework-cloud/framework-cloud-common/src/main/java/org/kie/cloud/common/logs/InstanceLogUtil.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.cloud.common.logs;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.kie.cloud.api.deployment.Deployment;
+import org.kie.cloud.api.deployment.Instance;
+
+public class InstanceLogUtil {
+
+    private static final String INSTANCES_LOGS_OUTPUT_DIRECTORY = "instance.logs";
+    private static final String DEFAULT_LOG_OUTPUT_DIRECTORY = "instances";
+    private static final String LOG_SUFFIX = ".log";
+
+    public static void writeInstanceLogs(Instance instance) {
+        File outputDirectory = new File(System.getProperty(INSTANCES_LOGS_OUTPUT_DIRECTORY, DEFAULT_LOG_OUTPUT_DIRECTORY));
```

Comment:
It would be good to separate logs by namespace. That way we can distinguish what logs belong to what test.